src/resources/views/crud/columns/table.blade.php

```diff
@@ -1,5 +1,12 @@
 @php
-	$value = data_get($entry, $column['name']);
+	if(!isset($column['value']))
+    {
+        $value = data_get($entry, $column['name']);
+    }
+    else
+    {
+        $value = $column['value'];
+    }
```

Comment:
I see no reason to not support this, thanks @rabol ! I think we can write this a lot shorter though, would using the null coalescing operator work just as well?
```suggestion
        $value = $column['value'] ?? data_get($entry, $column['name']);
```

Reply:
Oh I just thought of something for the maintenance team to check. 

For whoever picks up this issue: 

1) Do other columns support the `value` attribute? 
- If they do, I think we should add this.
- If they do NOT, I think we should talk about it some more before merging.

2) Which columns support the `value` attribute? If one column does, then... ALL columns should. Right?